modules/apps/object/object-api/src/main/java/com/liferay/object/field/business/type/ObjectFieldBusinessType.java

```diff
@@ -64,6 +64,14 @@ public default Object getDisplayContextValue(
 
 	public String getLabel(Locale locale);
 
+	public default Map<Locale, Object> getLocalizedValue(
```

Comment:
`Map` is used with variables in the plural form. Can you rename it to `getLocalizedValues`?

Reply:
Done!